### ISSUE TITLE
(Thunkable X) Controls in the block editor are not visible

### ISSUE THREAD

**Issue by AlexITMan** [thunkable/thunkable-issues#418]
Do you think I hid the trash can and zoom elements? No, but you can't see them.

![изображение](https://user-images.githubusercontent.com/20015046/75877296-68e8a000-5e28-11ea-87c4-c9623b762fc2.png)

**Comment by josmas** [comment]
They are under your blocks; you can see the bottom end of the trash can. Would you expect them to be on top? Is that what the report is about?

**Comment by AlexITMan** [comment]
Yes, if there are controls on the work field, they should be clearly visible. Why else would they be needed? It's like in a car: if the speedometer backlight doesn't work and it's very difficult to see the current speed, then why do you need a speedometer?
